resources/tests.py

```diff
@@ -0,0 +1,23 @@
+from django.test import TestCase
+from .models import Resource
+
+
+class TestResourceModels(TestCase):
+
+    def test_Resource_created_correctly(self):
+
+        resource1 = Resource.objects.create(name = 'Forking', link = "https://www.youtube.com/watch?v=HbSjyU2vf6Y")
+        resource2 = Resource.objects.create(name = 'The Git Story', link = "https://eventyret.github.io/the-git-story/")
+        resource1.save()
```

Comment:
Also when using `.objects.create()` you don't need to use `.save()`. The create method will automatically save it